src/mobile/src/main/java/tupperdate/android/chats/ConversationsPage.kt

```diff
@@ -0,0 +1,113 @@
+package tupperdate.android.chats
+
+import androidx.compose.material.Text
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyRowFor
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.ui.tooling.preview.Preview
+import tupperdate.android.ui.TupperdateTheme
+import tupperdate.api.RecipeApi
+import tupperdate.android.R
+import tupperdate.android.appbars.TupperdateTopBar
+import tupperdate.android.ui.components.ProfilePicture
+
+@Composable
+fun ConversationsPage(
+    onRecipeClick: () -> Unit,
+    onProfileClick: () -> Unit,
+    recipes: List<RecipeApi.Recipe>,
+    conversations: List<Conversation>,
+    modifier: Modifier = Modifier
+) {
+    Column(Modifier.fillMaxSize().padding(16.dp)) {
+        //TODO update TupperdateTopBar and fix the padding problem
+        TupperdateTopBar(
+            onChatClick = onRecipeClick,
+            onTitleClick = {},
+            onProfileClick = onProfileClick,
+            modifier = Modifier.fillMaxWidth()
+        )
+        Row(Modifier.padding(top = 26.dp, bottom = 16.dp)) {
+            Text(
+                text = stringResource(id = R.string.chat_matches_list),
+                style = MaterialTheme.typography.overline
+            )
+        }
+        Matches(recipes = recipes, modifier)
```

Comment:
Is there any reason why you give `modifier` to `Matches` instead of giving it to the `column`?

Reply:
No :)